code/modules/mob/living/death.dm

```diff
@@ -104,7 +104,12 @@
 
 /mob/living/carbon/death(gibbed)
 	. = ..()
-
+	//Death trauma
+	var/obj/item/organ/brain/B = getorgan(/obj/item/organ/brain)
+	if(istype(B) && B.organ_flags & ORGAN_VITAL)
+		var/trauma_type = pick(BRAIN_TRAUMA_MILD, BRAIN_TRAUMA_SEVERE)
+		gain_trauma_type(trauma_type, TRAUMA_RESILIENCE_MAGIC)
```

Comment:
Unremovable is `TRAUMA_RESILIENCE_ABSOLUTE`

Reply:
Thank you for pointing this out